Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+      description: The address group name.
+      type: String
+    - contextPath: PrismaSase.AddressGroup.description
+      description: The address group description.
+      type: String
+    - contextPath: PrismaSase.AddressGroup.addresses
+      description: The address group addresses.
+      type: String
+    - contextPath: PrismaSase.AddressGroup.dynamic_filter
+      description: The address group filter.
+      type: String
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
+      isArray: false
+      name: folder
+      predefined:
+      - Shared
+      - Mobile Users
+      - Remote Networks
+      - Service Connections
+      - Mobile Users Container
+      - Mobile Users Explicit Proxy
+      required: false
+    - auto: PREDEFINED
+      default: false
+      isArray: false
+      description: The address group type.
+      name: type
+      predefined:
+      - static
+      - dynamic
+      required: true
+    - default: false
+      description: static addresses for the address group. If the type is static, value must be provided.
+      isArray: false
+      name: static_addresses
+      required: false
+    - default: false
+      description: dynamic filter for the address group. If the type is dynamic, value must be provided.
+      isArray: false
+      name: dynamic_filter
+      required: false
+    - default: false
+      isArray: false
+      name: description
+      description: The address group's description.
+      required: false
+    - default: false
+      isArray: false
+      name: name
+      required: true
+      description: The name of the address group.
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    execution: false
+    description: Create a new address group.
+    name: prisma-sase-address-group-create
+    outputs:
+    - contextPath: PrismaSase.AddressGroup.id
+      description: The address group id.
+      type: String
+    - contextPath: PrismaSase.AddressGroup.name
+      description: The address group name.
+      type: String
+    - contextPath: PrismaSase.AddressGroup.folder
+      description: The address group folder.
+      type: String
+    - contextPath: PrismaSase.AddressGroup.description
+      description: The address group description.
+      type: String
+    - contextPath: PrismaSase.AddressGroup.addresses
+      description: The address group addresses.
+      type: String
+    - contextPath: PrismaSase.AddressGroup.dynamic_filter
+      description: The address group filter.
+      type: String
+  - arguments:
+    - default: false
+      isArray: false
+      name: group_id
+      description: The id of the address group.
+      required: true
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
```

```suggestion
      description: Prisma SASE folder location.
```
Is this the folder location or folder type/name?